ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java

```diff
@@ -66,6 +66,9 @@ public String getVersion() {
         LinearRaoParameters linearRaoParameters = parameters.getExtensionByName("LinearRaoParameters");
 
         preOptimSensitivityAnalysisResult = SystematicSensitivityAnalysisService.runAnalysis(network, crac, computationManager);
+        if (preOptimSensitivityAnalysisResult.getStateSensiMap().containsValue(null) && !crac.getRangeActions().isEmpty()) {
```

Comment:
You can add a comment to explain this test

Reply:
If the whole result is null nothing is done ?

Reply:
ok